pkg/asset/webhookconfiguration.go

```diff
@@ -88,9 +88,10 @@ func (m *mutatingWebhookConfiguration) New() *admissionregistrationv1beta1.Mutat
 						},
 					},
 				},
-				FailurePolicy:  &policy,
-				TimeoutSeconds: &timeoutSeconds,
-				SideEffects:    &sideEffects,
+				FailurePolicy:      &policy,
+				TimeoutSeconds:     &timeoutSeconds,
+				SideEffects:        &sideEffects,
+				ReinvocationPolicy: &reinvoke,
```

Comment:
how is this change related?

Reply:
I am not sure if this is directly needed for Min/Max but in general if the API server needs to re-invoke the admission webhook.

Reply:
it means that your admission webhook is recalled whenever one after you does a change. This is probably what you want. Just better make it conciously :)

Reply:
yeah, another mutating admission webhook after this one might add a new container. In this case, this webhook should have another shot at it.